src/common/utils.ts

```
@@ -198,3 +198,11 @@ export const getGameTypeName = (protoType: string, levelType: string) => {
 export const isGameTypeAudio = (protoType: string) => {
   return protoType !== 'Visible';
 }
+
+//Iterate and unsubscribe
+export const unsubscribeAll = (eventListeners: Function[] = []) : [] => {
+  eventListeners.forEach(unsubscribe => unsubscribe());
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)